gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -575,4 +584,16 @@ private static Project findDerivedProject(Project project, String projectName) {
             return project.getRootProject().findProject(projectName);
         }
     }
+
+    private static void configureEndpointMinimumVersions(
+            Project project, EndpointMinimumVersionsExtension minimumVersionsExtension) {
+        project.getPluginManager().apply(EndpointMinimumVersionsPlugin.class);
+        project.getTasks()
+                .named(
+                        EndpointMinimumVersionsPlugin.CONFIGURE_ENDPOINT_MINIMUM_VERSIONS_TASK,
+                        ConfigureEndpointMinimumVersionsTask.class,
+                        task -> {
+                            task.getVersions().set(minimumVersionsExtension.getEndpointVersions());
+                        });
```

Comment:
It's a bit confusing to me that we create the `EndpointMinimumVersionsExtension` in this plugin as well as in the `EndpointMinimumVersionsPlugin` plugin. Is there a reason for this?

Likewise we configure the `ConfigureEndpointMinimumVersionsTask` in this method and in the `EndpointMinimumVersionsPlugin`. Maybe we can get rid of one code path?

Reply:
yaaa... I have no idea why I did that.  I think too many looks at the recommended product dependencies configuration which is spread out in a few places.  Just applying the plugin should be enough.